libut/tests/Makefile

```diff
@@ -4,7 +4,8 @@ OBJS=$(patsubst %,%.o,$(PROGS))
 
 CFLAGS += -I../include
 CFLAGS += -g
-CFLAGS += -Wall -Wextra
+CFLAGS += -Wall -Wextra -Werror -fPIC
+CFLAGS += $(OPTFLAGS)
```

Comment:
These changes seem unwanted: I don't think `OPTFLAGS` is defined anywhere, and I'm not convinced that `-fPIC` is sufficiently portable/correct (as opposed to just letting the compiler do whatever it'd do by default).